bindata/v3.11.0/kube-controller-manager/svc.yaml

```diff
@@ -4,13 +4,12 @@ metadata:
   namespace: openshift-kube-controller-manager
   name: controller-manager
   annotations:
-    service.alpha.openshift.io/serving-cert-secret-name: serving-cert
     prometheus.io/scrape: "true"
-    prometheus.io/scheme: https
+    prometheus.io/scheme: http
```

Comment:
I expected us to remain https.  Why would we choose not to?

Reply:
1.12 has https in controller manager. Did we carry patch it in 3.11?

Reply:
we can wait for the 1.12 rebase to land maybe?

Reply:
1.12 is in. We can switch back.

Reply:
https?